src/megachatapi_impl.cpp

```diff
@@ -5650,7 +5650,27 @@ MegaChatMessagePrivate::MegaChatMessagePrivate(const Message &msg, Message::Stat
                 }
 
                 priv = callEndInfo->duration;
-                code = callEndInfo->termCode;
+                switch(callEndInfo->termCode)
+                {
+                    case END_CALL_REASON_CANCELLED:
```

Comment:
If this reason for a call termination is obsolete, we should reflect it in the documentation